solutions/ac-MultipleOf2019.mdx

```diff
@@ -2,11 +2,33 @@
 id: ac-MultipleOf2019
 source: AtCoder Beginner Contest
 title: Multiple of 2019
-author: Maggie Liu, Brad Ma, Mohammad Nour Massri
+author: Maggie Liu, Brad Ma, Mohammad Nour Massri, David Zhang
 ---
 
 [Official Analysis](https://img.atcoder.jp/abc164/editorial.pdf#page=10)
 
+## Explanation
+Firstly, in order to store sections of the string, we can make a suffix array with the condition that at index $$i$$ in suffix array $${S}$$
+
+$${S_i = s_n + 10*s_{n-1} +100*s_{n-2} + ... + {10^{n-i}} * s_{i}}$$
+
+For example, in the string "1234," the suffix array would be
```

Comment:
is the comma supposed to be in the quotes?

Reply:
no lol